Integrations/AlienVault_USM_Anywhere/AlienVault_USM_Anywhere.py

```diff
@@ -199,6 +219,22 @@ def parse_events(events_data):
     return events
 
 
+def dict_value_to_int(target_dict: Dict, key: str):
+    """
+    :param target_dict: A dictionary which has the key param
+    :param key: The key that we need to convert it's value to integer
+    :return: The integer representation of the key's value in the dict params
+    """
+    try:
+        if target_dict:
+            value = target_dict.get(key)
+            if value:
+                target_dict[key] = int(value)
+                return target_dict[key]
+    except ValueError:
+        raise ValueError(f'This value for {key} must be an integer.')
```

Comment:
you did not fix this one...
```suggestion
        raise ValueError(f'The value for {key} must be an integer.')
```